plugins/integrations/kubernetes-service/src/main/java/com/cloud/kubernetes/cluster/actionworkers/KubernetesClusterStartWorker.java

```diff
@@ -571,6 +571,13 @@ public boolean startKubernetesClusterOnCreate() {
         if (!isKubernetesClusterDashboardServiceRunning(true, startTimeoutTime)) {
             logTransitStateAndThrow(Level.ERROR, String.format("Failed to setup Kubernetes cluster : %s in usable state as unable to get Dashboard service running for the cluster", kubernetesCluster.getName()), kubernetesCluster.getId(),KubernetesCluster.Event.OperationFailed);
         }
+        retrieveScriptFiles();
+        copyScripts(publicIpAddress, sshPort);
```

Comment:
What about creating a method for these lines on `KubernetesClusterActionWorket` and invoke it here and on `KubernetesClusterActionWorket` instead of lines 475-482